src/ale_interface.hpp

```diff
@@ -143,6 +143,9 @@ class ALEInterface {
   //followed by the green colours and then the blue colours
   void getScreenRGB(std::vector<unsigned char>& output_rgb_buffer);
 
+  //This method should receive an empty vector to fill it with audio data
```

Comment:
Missed this old comment (change to "Returns the current audio data"). Will do on next commit.